sktime/utils/validation/forecasting.py

```diff
@@ -271,6 +271,8 @@ def check_fh(fh, enforce_relative=False):
         Forecasting horizon specifying the time points to predict.
     enforce_relative : bool, optional (default=False)
         If True, checks if fh is relative.
+    freq : str, optional (default=None)
+        Frequency string
```

Comment:
please add details to the docstring how the parameters influence the output, and what errors are raised

Reply:
details are added